frontstage/templates/surveys/surveys.html

```diff
@@ -54,12 +54,14 @@
             <span id="status-{{ loop.index }}">{{ survey.status }}</span>
         </div>
         <div class="grid__col col-2@m">
+            {% if not survey.status == 'Complete' or survey.case.caseEvents|selectattr('category', 'equalto', 'EQ_LAUNCH')|list|length == 0 %}
```

Comment:
survey.collection_instrument_type, holds the type if it be EQ or SEFT, so this isn't needed

Reply:
LOL yeah Tej mentioned that today after him and Andrew told me Friday you had to look in case events :-P I've updated the code accordingly

Reply:
Forgot that was there 👀